apps/pipelines/nodes/nodes.py

```diff
@@ -398,6 +398,10 @@ class LLMResponseWithPrompt(LLMResponse, HistoryMixin, OutputMessageTagMixin):
         PipelineChatHistoryTypes.GLOBAL,
         json_schema_extra=UiSchema(widget=Widgets.history, enum_labels=PipelineChatHistoryTypes.labels),
     )
+    voice_provider_id: OptionalInt = Field(
```

Comment:
I'm not quite following why we need this field in the node since the value is always going to be `experiment.voice_provider_id`. It seems like we can just drop this field and only record the voice ID.

Am I missing something?